pkg/controller/replication_controller.go

```diff
@@ -178,11 +205,13 @@ func (rm *ReplicationManager) watchControllers(resourceVersion *string) {
 	}
 }
 
-// Helper function. Also used in pkg/registry/controller, for now.
-func FilterActivePods(pods []api.Pod) []api.Pod {
+// filterActivePodsByNamespace returns all 'Active' pods in the given namespace.
+func filterActivePodsByNamespace(namespace string, pods []api.Pod) []api.Pod {
```

Comment:
If we just use len() on the array - we should just return the number of them, without copying anything (not even return their names).